### ISSUE TITLE
Container Running OOM on Unraid

### ISSUE THREAD

**Issue by Trozmagon** [Josh5/unmanic#161]
### Description
Using any other container tag except for 0.0.1 there seems to be a memory leak causing Unraid to kill it. There's nothing in the unmanic logs only in the syslog of Unraid when it kills the contianer:

- server kernel: Memory cgroup out of memory: Killed process 37376 (unmanic) total-vm:16305576kB, anon-rss:3710628kB, file-rss:0kB, shmem-rss:3312kB, UID:99 pgtables:16744kB oom_score_adj:0

### Steps to Reproduce
Using container tag :latest or :staging or :0.0.2 and the same settings as: [settings.txt](https://github.com/Josh5/unmanic/files/6206126/settings.txt) start a single job and it will crash sometime during the task.

**Expected behavior:** Task to complete and move on to next task.
**Actual behaviour:** Container runs out of memory and gets killed by OS
**Reproduces how often:** Every time.

### Versions
0.0.1+63e0bca
Unraid version 6.9.1

### Installation Method Used
Docker

### Additional Information
- Cache is mapped to an SSD so it is not transcoding into RAM.
- Only have 1 worker process.
- Using Nvidia Driver version 460.67 with a GTX 1650
- It was running some time ago without issues on another server I had with an RTX 3080 in January (This is probably a red herring as there's been unraid/nvidia drivers/unmanic updates since)
- "Enable HW Decoding" doesn't seem to affect it

There seems to be something being dumped into the docker image, I've seen it hit 100% usage and then once killed will come back down to 60% (usual usage size).

Unmanic Log: [unmanic.log](https://github.com/Josh5/unmanic/files/6205860/unmanic.log)

![image](https://user-images.githubusercontent.com/18479642/112495707-0b4d2b80-8d7c-11eb-83fa-7e116e739fc0.png)
![image](https://user-images.githubusercontent.com/18479642/112498896-dd1d1b00-8d7e-11eb-8414-5845c7ff58e1.png)
![image](https://user-images.githubusercontent.com/18479642/112504973-3cc9f500-8d84-11eb-90f3-1244fbff759a.png)
![image](https://user-images.githubusercontent.com/18479642/112506475-b6aeae00-8d85-11eb-8714-c426de451eeb.png)

**Comment by Josh5** [comment]
Could you give me your unraid configs?

**Comment by Trozmagon** [comment]
Sorry mate, my github notification email went to my spam folder so I'm only just seeing your comment now. I'll send you over my config later today.

I have managed to narrow this down a bit, I was able to get it to work when I removed my TV Show library and just had my Movies library configured which meant it went from checking 20k files down to 1.3k so this could be related to the adding files to the pending tasks script.

**Comment by Josh5** [comment]
Do you have the library event watcher enabled? This can crash and cause all kinds of a mess when there are too many files in your library

**Comment by Trozmagon** [comment]
Yeah you've just nailed it, it was on and after I switched it off and re-added my TV library there were no more issues. Thanks for your help mate and great work on this!
